com/edocti/jintro/lab01/Operators2.java

```diff
@@ -0,0 +1,56 @@
+package com.edocti.jintro.lab01;
+import static java.lang.System.*;
+
+public class Operators2 {
+    
+	public static String toBinary (int n) {
+        // return Integer.toBinaryString(n);
+        return String.format("%32s", Integer.toBinaryString(n)).replace(' ', '0');
+    }
+    
+	public static void main (String[] args) {
+        int i = 10;
+        out.println(i++); // i=i+1
+        out.println(i); // 11
+        out.println(++i);
+        
+		i = 10;
+        out.println(toBinary(i));
+        out.println(toBinary(i << 1)); // 0000 0010 => 00000100
+        
+		out.println(i << 1); // valoare de dinainte inmultita cu 2
```

Comment:
same here (maybe you're using both tabs and spaces and tab size is bigger than you think)